test/ISWAPROUTER12.t.sol

```diff
@@ -1,92 +1,78 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 import "./ISWAPROUTER12.sol";
-import "truffle/console.sol";
-
 contract ISwapRouter12Test {
-    ISwapRouter12 public router;
-
+    ISwapRouter12 public router;(

```

Comment:
Hey abemelek the required test was supposed to be for a contract and not the interface. Also kirubels swap contract is the one that was supposed to be tested